Plugin: 摩尔质量计算器

**插件名称：**

摩尔质量计算器

**插件功能：**

帮助中学生计算摩尔质量或相对分子质量的小工具。

**PyPI 项目名：**

nonebot-plugin-molar-mass

**插件 import 包名：**

nonebot_plugin_molar_mass

**插件项目仓库/主页链接：**

https://github.com/kifuan/nonebot-plugin-molar-mass

**标签：**

[]

# 📃 商店发布检查结果

> Plugin: 摩尔质量计算器

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-molar-mass 的信息如下：
    name         : nonebot-plugin-molar-mass                                  
     version      : 0.1.1                                                      
     description  : A tool to calculate molar mass for middle school students. 
    
    dependencies
     - nonebot2 &gt;=2.0.0rc3,&lt;3.0.0
     - rply &gt;=0.7.8,&lt;0.8.0
插件 nonebot_plugin_molar_mass 加载出错：
01-22 16:24:20 [SUCCESS] nonebot | NoneBot is initializing...
01-22 16:24:20 [INFO] nonebot | Current Env: prod
01-22 16:24:20 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_molar_mass&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_molar_mass&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_molar_mass/__init__.py&quot;, line 3, in &lt;module&gt;
    from nonebot.adapters.onebot.v11.message import Message
ModuleNotFoundError: No module named &#x27;nonebot.adapters.onebot&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 项目 <a href="https://github.com/kifuan/nonebot-plugin-molar-mass">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-molar-mass/">nonebot-plugin-molar-mass</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -3856,5 +3856,15 @@
     "homepage": "https://github.com/longchengguxiao/nonebot_plugin_pvz",
     "tags": [],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_molar_mass",
+    "project_link": "nonebot-plugin-molar-mass",
+    "name": "摩尔质量计算器",
+    "desc": "帮助中学生计算摩尔质量或相对分子质量的小工具。",
+    "author": "kifuan",
+    "homepage": "https://github.com/kifuan/nonebot-plugin-molar-mass",
+    "tags": [],
+    "is_official": false
   }
 ]

